discovery/linode/linode_test.go

```diff
@@ -177,6 +179,169 @@ func TestLinodeSDRefresh(t *testing.T) {
+			"__meta_linode_region":               model.LabelValue("us-east"),
+			"__meta_linode_type":                 model.LabelValue("g6-standard-2"),
+			"__meta_linode_status":               model.LabelValue("running"),
+			"__meta_linode_tags":                 model.LabelValue(",monitoring,"),
+			"__meta_linode_group":                model.LabelValue(""),
+			"__meta_linode_gpus":                 model.LabelValue("0"),
+			"__meta_linode_hypervisor":           model.LabelValue("kvm"),
+			"__meta_linode_backups":              model.LabelValue("disabled"),
+			"__meta_linode_specs_disk_bytes":     model.LabelValue("85899345920"),
+			"__meta_linode_specs_memory_bytes":   model.LabelValue("4294967296"),
+			"__meta_linode_specs_vcpus":          model.LabelValue("2"),
+			"__meta_linode_specs_transfer_bytes": model.LabelValue("4194304000"),
+			"__meta_linode_extra_ips":            model.LabelValue(",96.126.108.16,192.168.201.25,"),
+		},
+		{
+			"__address__":                        model.LabelValue("66.228.47.103:80"),
+			"__meta_linode_instance_id":          model.LabelValue("26837992"),
+			"__meta_linode_instance_label":       model.LabelValue("prometheus-linode-sd-exporter-4"),
+			"__meta_linode_image":                model.LabelValue("linode/ubuntu20.04"),
+			"__meta_linode_private_ipv4":         model.LabelValue("192.168.148.94"),
+			"__meta_linode_public_ipv4":          model.LabelValue("66.228.47.103"),
+			"__meta_linode_public_ipv6":          model.LabelValue("2600:3c03::f03c:92ff:fe1a:fb4c"),
+			"__meta_linode_private_ipv4_rdns":    model.LabelValue(""),
+			"__meta_linode_public_ipv4_rdns":     model.LabelValue("li328-103.members.linode.com"),
+			"__meta_linode_public_ipv6_rdns":     model.LabelValue(""),
+			"__meta_linode_region":               model.LabelValue("us-east"),
+			"__meta_linode_type":                 model.LabelValue("g6-nanode-1"),
+			"__meta_linode_status":               model.LabelValue("running"),
+			"__meta_linode_tags":                 model.LabelValue(",monitoring,"),
+			"__meta_linode_group":                model.LabelValue(""),
+			"__meta_linode_gpus":                 model.LabelValue("0"),
+			"__meta_linode_hypervisor":           model.LabelValue("kvm"),
+			"__meta_linode_backups":              model.LabelValue("disabled"),
+			"__meta_linode_specs_disk_bytes":     model.LabelValue("26843545600"),
+			"__meta_linode_specs_memory_bytes":   model.LabelValue("1073741824"),
+			"__meta_linode_specs_vcpus":          model.LabelValue("1"),
+			"__meta_linode_specs_transfer_bytes": model.LabelValue("1048576000"),
+			"__meta_linode_extra_ips":            model.LabelValue(",172.104.18.104,"),
+			"__meta_linode_ipv6_ranges":          model.LabelValue(",2600:3c03:e000:123::/64,"),
+		},
+	} {
+		t.Run(fmt.Sprintf("item %d", i), func(t *testing.T) {
+			require.Equal(t, lbls, tg.Targets[i])
+		})
+	}
+}
+func TestLinodeSDRefreshRegionCaCentral(t *testing.T) {
```

Comment:
Having an individual test function for each region, when they're both testing the same functionality, and where the only differences are the inputs and outputs, feels like it should be a single test IMO

Reply:
I wasn't sure how to make a test that would emulate multiple queries, hence three tests for two different regions and a non region filtered query.

Reply:
Moving the constants out to their own file greatly improved the readability here, does that satisfy?  To me it felt that 3 distinct tests made more sense.

Reply:
Assuming we move this test suite to something closer to table driven tests, we could move the region config into a field for each variation of the tests. We would also need to move much of the discovery setup/configuration into the actual loop of the test table so that we can configure it time with the expected region result. It'd be a fairly large refactor of the test suite here, but it should be possible. 

> To me it felt that 3 distinct tests made more sense.

I disagree -- all three tests are still testing the same functionality; they should all be part of a test suite for the same test, IMO.

Reply:
Converted to TableDrivenTests in 145c7858d45b63b4a1ae53972240dca3afe154dc